File: src/main/java/module/teamoftheweek/TeamOfTheWeekModule.java
package module.teamoftheweek;

import core.model.TranslationFacility;
import core.module.DefaultModule;
import module.teamoftheweek.gui.TeamOfTheWeekPanel;

import javax.swing.*;
import java.awt.event.KeyEvent;

public class TeamOfTheWeekModule extends DefaultModule {

	@Override
	public KeyStroke getKeyStroke() {
		return KeyStroke.getKeyStroke(KeyEvent.VK_3, KeyEvent.CTRL_MASK);
	}

	@Override
	public int getModuleId() {
		return TEAM_OF_THE_WEEK;
	}

	@Override
	public String getDescription() {
		return TranslationFacility.tr("Tab_TeamOfTheWeek");
	}

	@Override
	public JPanel createTabPanel() {
		return new TeamOfTheWeekPanel();
	}

}

File: src/main/java/module/teamOfTheWeek/TeamOfTheWeekModule.kt
package module.teamOfTheWeek;

import java.awt.event.KeyEvent;

import javax.swing.JPanel;
import javax.swing.KeyStroke;


import core.model.HOVerwaltung;
import core.module.DefaultModule;
import module.teamOfTheWeek.gui.TeamOfTheWeekPanel;

public class TeamOfTheWeekModule extends DefaultModule {

	@Override
	public KeyStroke getKeyStroke() {
		return KeyStroke.getKeyStroke(KeyEvent.VK_3, KeyEvent.CTRL_MASK);
	}

	@Override
	public int getModuleId() {
		return TEAM_OF_THE_WEEK;
	}

	@Override
	public String getDescription() {
		return HOVerwaltung.instance().getLanguageString("Tab_TeamOfTheWeek");
	}

	@Override
	public JPanel createTabPanel() {
		return new TeamOfTheWeekPanel();
	}

}
